### DISCUSSION TITLE
cdn به غیر از cloudflare

### DISCUSSION THREAD

**Discussion by salaryy** [hiddify/Hiddify-Manager#488] [Q&A سوالات رایج]
دورود و خسته نباشید 

با توجه به شرایط فعلی فقط سویس cdn جواب گو هست و بقیه سرویس ها تقریبا بی فایده هستن (پینگ بسیار بالا)
سرویس آوران فقط پورت 80 قابل استفاده است که امن نیست و در خیلی موارد EOF داره کلا روی cdn لینک سرور لود نمیشه
سرویس های خارجی مثل cdn77 , bunny پنل ادمین لود میشه ولی WS کار نمیکنه و دیتایی جابه جا نمیشه
سرویس کلاد فلیر هم تقریبا فیلتر شده
سوال اولم راهکاری برای پورت 443 اوران وجود داره؟
سرویس خارجی به غیر از cloudflare  چی کار میکنه؟

**Comment by salaryy** [comment]
هیچ کس نبود بگه مشکل EOF  در سایت های https در cdn هارو چه طوری رفع کنیم؟

**Reply by mohsen-armoun** [reply]
گاهی مشکل از تنظیات ساعت کلاینت و سرور هست، اختلاف ساعت بینشون رو چک کنین، یا یک ساعت ،ساعت کلاینت رو جلو ببرین یا عقب

**Comment by salaryy** [comment]
![image](https://user-images.githubusercontent.com/26680104/229278664-c83f4d20-1f57-4c63-b190-2b0baa4a9db4.png)

**Reply by SajadQamari** [reply]
این احتمالا بخاطر سرتیفیکته
